packages/web3-utils/src/errors.ts

```diff
@@ -0,0 +1,22 @@
+export const errorMessage = (value: unknown, error: string) =>
```

Comment:
I think errors should remain uniformed to how they were done [here](https://github.com/ChainSafe/web3.js/blob/4.x/packages/web3-common/src/errors.ts)